packages/react-environment/src/api-providers/web3/web3-mixer-deposit.ts

```diff
@@ -48,8 +46,7 @@ export class Web3MixerDeposit extends MixerDeposit<WebbWeb3Provider, DepositPayl
     };
   }
 
-  async getSizes(contract: string = 'nativeAnchor'): Promise<MixerSize[]> {
-    const storage = await this.inner.chainStorage;
-    return storage[contract].mixerSize;
+  async getSizes(): Promise<MixerSize[]> {
+    return this.inner.getMixersSizes(WebbWeb3Provider.getNativeCurrencySymbol(await this.inner.getChainId()));
```

Comment:
I think you changed the name of this function above to `getMixersInfo` no? Nonetheless, we should call it `getMixerSizes` as was brought up at the function def.

Reply:
I've renamed the functions and type to getMixerTitles and MixerTitle respectively.

Reply:
On the same topic, please remove MixersXYZs naming convention. The plural should not be applied twice. It should be MixerXYZs.

For mixer titles, it should be getMixerTitles not getMixersTitles. The latter is more awkward.

For example, for something more semantically clear, if I had an asset pallet where each asset had a name, I wouldn't want to name things as getAssetsNames, I'd want to say getAssetNames which is similar to getAllAssetNames. So semantically, we're looking for "get me all mixer sizes" not "get me all mixers sizes". Do you agree or disagree?